alembic/versions/c819d65fd831_add_news_flash_qualification.py

```diff
@@ -0,0 +1,37 @@
+"""add news flash qualification
+
+Revision ID: c819d65fd831
+Revises: fe6dcd0178f7
+Create Date: 2022-08-05 21:03:00.853536
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "c819d65fd831"
+down_revision = "fe6dcd0178f7"
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+from anyway.backend_constants import BE_CONST, NewsflashLocationQualification
+
+
+def upgrade():
+    op.add_column(
+        "news_flash",
+        sa.Column(
+            "newsflash_location_qualification",
+            sa.Integer(),
+            server_default=sa.text(f"{NewsflashLocationQualification.NOT_VERIFIED.value}"),
```

Comment:
Does this mean the values for existing records, and new records where the field is not explicitly set will be set to NOT_VERIFIED?

Reply:
in this area of the code the server_default will add a default value for all the previous existing rows.
for new recored to have a default value, there is another place where i used server_default.